model_analyzer/config/generate/perf_analyzer_config_generator.py

```diff
@@ -154,87 +152,103 @@ def set_last_results(
             measurement = [max(valid_measurements)]
 
             self._last_results = measurement
-            self._concurrency_results.extend(measurement)
+            self._parameter_results.extend(measurement)
 
-    def _create_concurrency_list(self, cli_config: ConfigCommandProfile,
-                                 model_parameters: dict) -> List[int]:
+    def _create_parameter_list(self, model_parameters: dict) -> List[int]:
         if model_parameters['concurrency']:
             return sorted(model_parameters['concurrency'])
-        elif cli_config.run_config_search_disable:
+        elif self._cli_config.request_rate_range:
```

Comment:
Why do we look at model_parameters for concurrency but cli for rrr? Where does model_parameters come from?

Reply:
I've added request rate to model parameters